openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -35,9 +36,15 @@ class ArnoldStandinLoader(load.LoaderPlugin):
     color = "orange"
 
     def load(self, context, name, namespace, options):
+        if not cmds.pluginInfo("mtoa", query=True, loaded=True):
+            # Allow mtoa plugin load to process all its events
+            # because otherwise `defaultArnoldRenderOptions.operator`
+            # does not exist yet and some connections to the standin
+            # can't be correctly generated on create resulting in an error
+            cmds.loadPlugin("mtoa")
+            cmds.refresh(force=True)
+            maya.utils.processIdleEvents()
```

Comment:
```suggestion
            cmds.loadPlugin("mtoa")
            createOptions()
```
You can create the default Arnold nodes, which resolves this issue. Remember to `from mtoa.core import createOptions` at the top.
https://arnoldsupport.com/2015/12/09/mtoa-creating-the-defaultarnold-nodes-in-scripting/